src/test/java/org/opensearch/sdk/TestExtensionsRunner.java

```diff
@@ -95,6 +99,17 @@ public void setUp() throws Exception {
         );
     }
 
+    @Override
+    @AfterEach
+    public void tearDown() throws Exception {
+        super.tearDown();
```

Comment:
Thanks for adding the `tearDown`. This will definitely avoid the bind connection error.